e2e/ls_test.go

```diff
@@ -391,3 +393,204 @@ func TestListS3ObjectsWithDashH(t *testing.T) {
 		1: match(`^ 264.0K testfile2.txt$`),
 	}, trimMatch(dateRe), alignment(true))
 }
+
+// ls --exclude "main*" s3://bucket/
+func TestListS3ObjectsWithExcludeFilter(t *testing.T) {
+	t.Parallel()
+
+	bucket := s3BucketFromTestName(t)
+
+	const (
+		content        = "content"
+		excludePattern = "*.txt"
+	)
```

Comment:
Pattern in comment is different from the one defined in test.

Reply:
changed the pattern in other [PR](https://github.com/peak/s5cmd/pull/334).